pyCFOFiSAX/_tree_iSAX.py

```diff
@@ -264,29 +264,29 @@ def insert(self, new_sequence):
 
     def insert_paa(self, new_paa):
         """
-        La fonction insert qui appelle directement celle de son nœud root
+        The insert function that directly calls that of its root node
```

Comment:
The `insert` function that directly calls the function of its root node

Reply:
```suggestion
        The insert function that directly calls the function of its root node
```